content/_help/manage-your-account/overview._en.md

```diff
@@ -6,25 +6,15 @@ permalink: /help/manage-your-account/overview/
 meta_title: Manage your account
 order: 0
 redirect_from:
+  - /en/help/manage-your-account/
+  - /en/help/manage-your-account/overview/
   - /help/changing-settings/add-or-remove-email-address/
```

Comment:
Maybe we can redirect this to https://login.gov/help/manage-your-account/change-your-email-address/ ?